File: app/src/main/java/com/example/weatherid/Models/Clouds.java
package com.example.weatherid.Models;

public class Clouds{
	private double all;

	public void setAll(int all){
		this.all = all;
	}

	public double getAll(){
		return all;
	}

	@Override
 	public String toString(){
		return 
			"Clouds{" + 
			"all = '" + all + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/example/weatherid/Models/Clouds.kt
package com.example.weatherid.Models

class Clouds {
    var all = 0.0
        private set

    fun setAll(all: Int) {
        this.all = all.toDouble()
    }

    override fun toString(): String {
        return "Clouds{" +
                "all = '" + all + '\'' +
                "}"
    }
}
